src/clusterStatusCheck.ts

```diff
@@ -1,33 +1,18 @@
 import {ClusterStatusResponse, OldestNodeResponse} from './utils/handlerInputs';
 import {getClusterHealth} from './elasticsearch/elasticsearch';
-import {ElasticsearchClusterStatus} from './elasticsearch/types';
-import {describeAsg} from "./aws/autoscaling";
-import {AutoScalingGroupsType} from "aws-sdk/clients/autoscaling";
 
 export async function handler(event: OldestNodeResponse): Promise<ClusterStatusResponse> {
-    return new Promise<ClusterStatusResponse>((resolve, reject) => {
-        Promise.all([
-            describeAsg(event.asgName),
-            getClusterHealth(event.oldestElasticsearchNode.ec2Instance.id)
```

Comment:
Moved this to `autoScalingGroupCheck` since we have access to ASG info here  so can avoid the extra call